<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>2029</width>
    <height>1287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:purple;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>1181</width>
      <height>1101</height>
     </rect>
    </property>
    <property name="title">
     <string>Gestion des investisseurs</string>
    </property>
    <widget class="QLabel" name="nom_gestion">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>50</y>
       <width>271</width>
       <height>61</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Cambria</family>
       <pointsize>28</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:#ffc778;
border-radius: 10px;</string>
     </property>
     <property name="text">
      <string>Investisseurs</string>
     </property>
    </widget>
    <widget class="QGroupBox" name="afficher">
     <property name="geometry">
      <rect>
       <x>730</x>
       <y>130</y>
       <width>411</width>
       <height>441</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 12px;
 background-color:#be7fbe;
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 10px;
}

QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

QLabel {
    font-weight: bold;
    padding: 1px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="afficher_label">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>291</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>14</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Liste des investissements</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableInvest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>90</y>
        <width>391</width>
        <height>301</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">QTableWidget{
border-radius: 10px;
background-color:#D6DCE0;

}</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdf">
      <property name="geometry">
       <rect>
        <x>270</x>
        <y>50</y>
        <width>131</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">
QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

</string>
      </property>
      <property name="text">
       <string>Export PDF</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icones/Icones/pdf.png</normaloff>:/Icones/Icones/pdf.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>30</width>
        <height>40</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
</string>
      </property>
      <property name="text">
       <string>TRI par date</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icones/Icones/ajouter.png</normaloff>:/Icones/Icones/ajouter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <zorder>afficher_label</zorder>
     <zorder>pdf</zorder>
     <zorder>ajouter_5</zorder>
     <zorder>tableInvest</zorder>
    </widget>
    <widget class="QGroupBox" name="Delete">
     <property name="geometry">
      <rect>
       <x>370</x>
       <y>140</y>
       <width>321</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 12px;
 background-color:#be7fbe;
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 10px;
}

QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

QLabel {
    font-weight: bold;
    padding: 1px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="supprimer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>10</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Supprimer un investissement</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Date</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom_label_2">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>60</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Nom</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="DeleteName">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>60</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>310</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
</string>
      </property>
      <property name="text">
       <string>Supprimer</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icones/Icones/supprimer.png</normaloff>:/Icones/Icones/supprimer.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QDateEdit" name="DeleteDate">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="nom_label_8">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>190</y>
        <width>321</width>
        <height>51</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Voulez-vous vraiment supprimer cet investissement?</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="confirm">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>270</y>
        <width>91</width>
        <height>31</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#ffc778;;
</string>
      </property>
      <property name="text">
       <string>Confirmer</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Add">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>331</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 12px;
 background-color:#be7fbe;
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 10px;
}

QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

QLabel {
    font-weight: bold;
    padding: 1px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QDateEdit" name="date_Invest">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>160</y>
        <width>110</width>
        <height>22</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="dateLbael">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>130</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Date d'investissement</string>
      </property>
     </widget>
     <widget class="QLabel" name="descLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>280</y>
        <width>231</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Description de l'investissement</string>
      </property>
     </widget>
     <widget class="QLabel" name="Ajouter">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Ajouter un investissement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nom_Inv">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>80</y>
        <width>131</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>151</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Nom d'invetisseur</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_add">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>400</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
</string>
      </property>
      <property name="text">
       <string>Ajouter</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icones/Icones/ajouter.png</normaloff>:/Icones/Icones/ajouter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
     <widget class="QLabel" name="budgetLabel">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>200</y>
        <width>201</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Budget d'investissement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="budget_Invest">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>230</y>
        <width>141</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="desc_Invest">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>310</y>
        <width>311</width>
        <height>71</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="Modifier">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>610</y>
       <width>331</width>
       <height>431</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QGroupBox {
    font: 12px;
 background-color:#be7fbe;
}

QLineEdit {
    border: 2px solid gray;
    border-radius: 10px;
    padding: 10px;
}

QPushButton {
    background-color: rgb(58, 195, 197);
    color: white;
    border-radius: 5px;
    padding: 5px;
}

QLineEdit:focus, QPushButton:focus {
    border: 2px solid #4CAF50;
}

QLabel {
    font-weight: bold;
    padding: 1px;
}</string>
     </property>
     <property name="title">
      <string/>
     </property>
     <widget class="QLabel" name="Modifier_2">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>10</y>
        <width>221</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Cambria</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <italic>false</italic>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
border-radius: 10px;</string>
      </property>
      <property name="text">
       <string>Modifier un investissement</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="searchID">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>100</y>
        <width>161</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
     </widget>
     <widget class="QLabel" name="id_label_4">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>171</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">border-radius: 10px;
background-color:#D6DCE0;
</string>
      </property>
      <property name="text">
       <string>Id de l'investissement</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_search">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>220</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="maximumSize">
       <size>
        <width>101</width>
        <height>31</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:#ffc778;
</string>
      </property>
      <property name="text">
       <string>Chercher</string>
      </property>
      <property name="icon">
       <iconset>
        <normaloff>:/Icones/Icones/ajouter.png</normaloff>:/Icones/Icones/ajouter.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>20</width>
        <height>20</height>
       </size>
      </property>
     </widget>
    </widget>
    <zorder>afficher</zorder>
    <zorder>nom_gestion</zorder>
    <zorder>Delete</zorder>
    <zorder>Add</zorder>
    <zorder>Modifier</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2029</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
